docs/tutorial/vald-agent-standalone-on-k8s.md

```diff
@@ -284,21 +284,23 @@ This chapter uses [NGT](https://github.com/yahoojapan/ngt) as Vald Agent to perf
 
             </details>
 
-        - [Optional] Indexing manually instead of waiting for auto indexing
+        - [Optional] Indexing manually instead of waiting for auto indexing.
```

Comment:
📝 I fixed the indentation because it is broken.
<img width="350" alt="img" src="https://github.com/vdaas/vald/assets/25459661/b7e3cf88-9285-4655-9830-548ef1c34134">

👉 https://vald.vdaas.org/docs/tutorial/vald-agent-standalone-on-k8s/